.github/workflows/test.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           allow-prereleases: true
 
       - name: Install uv
-        uses: astral-sh/setup-uv@v5
+        uses: astral-sh/setup-uv@v6
         with:
           cache-dependency-glob: "**/pyproject.toml"
```

Comment:
Now part of the default: https://github.com/astral-sh/setup-uv#cache-dependency-glob
```suggestion
```